File: mapintegrationlib/src/main/java/org/unyde/mapintegrationlib/database/MyDatabase.java
package org.unyde.mapintegrationlib.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import org.unyde.mapintegrationlib.database.dao.*;
import org.unyde.mapintegrationlib.database.entity.*;

@Database(entities = {PathNode.class, PathNode_connet.class, Cluster_Primary_Info.class, CheckInCheckOut.class, MallMapMain.class}, version = 1)


public abstract class MyDatabase extends RoomDatabase {

    public abstract PathNodeDao pathNodeList();

    public abstract PathNodeConnectDao pathNodeConnectList();

    public abstract ClusterPrimaryInfoDao clusterPrimaryInfo();

    public abstract CheckinCheckOutDao checkInCheckOut();

    public abstract MallMapMainDao mallMapMain();

}

File: app/src/main/java/org/unyde/mapintegration/database/MyDatabase.kt
package org.unyde.mapintegration.database


import androidx.room.Database
import androidx.room.RoomDatabase
import org.unyde.mapintegration.database.dao.PathNodeConnectDao
import org.unyde.mapintegration.database.dao.PathNodeDao
import org.unyde.mapintegration.database.entity.PathNode
import org.unyde.mapintegration.database.entity.PathNode_connet

@Database(entities = [PathNode::class, PathNode_connet::class], version = 1)
abstract class MyDatabase : RoomDatabase() {

    abstract fun pathNodeList(): PathNodeDao

    abstract fun pathNodeConnectList(): PathNodeConnectDao

}
